src/base/bootstrap-overrides/_base.scss

```diff
@@ -589,6 +589,35 @@ input[type=button], input[type=reset], input[type=submit] {
 	width: auto;
 }
 
+/*
+ * Explicit labels for checkbox and radio button inputs
+ *
+ * Allow the use of horizontal explicit labels instead of the default implicit
+ * labelling pattern for checkbox and radio button inputs provided by Bootstrap.
+ */
+.form-inline {
+	.label-inline {
+		position: relative;
+		vertical-align: middle;
+		label {
+			font-weight: 400;
+			margin-bottom: 0px;
+			padding-left: 2px;
+		}
+	}
+}
+@media (min-width: $screen-sm-min) {
```

Comment:
Move this in the view or create one for bootstrap and add it the main SCSS include file